src/actions/userActions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { SET_SIGNUP_FORM_INPUT, SET_SUBMIT_USERS, USER_ERRORS } from './type';
```

Comment:
It is good practice to consider success and a failure of the action when naming types. In this case, you are registering a user it would make sense if your types are is `REGISTER_USER_SUCCESS` for successful registration and `REGISTER_USER_FAILURE` for  an unsuccessful registration.

Reply:
Thanks for the feedback, I agree with you, let me rename that.